src/main/java/com/celements/web/plugin/cmd/ExternalJavaScriptFilesCommand.java

```diff
@@ -188,11 +192,19 @@ public String getAllExternalJavaScriptFiles() throws XWikiException {
           context.getDatabase(),
           context.getDoc().getDocumentReference().getLastSpaceReference().getName(),
           "WebPreferences"), context));
-      addAllExtJSfilesFromDoc(context.getDoc());
-      XWikiDocument pagetype = getPageTypeDoc(context.getDoc());
-      if (pagetype != null) {
-        addAllExtJSfilesFromDoc(pagetype);
+      PageTypeReference pageTypeRef = getPageTypeResolver().getPageTypeRefForDocWithDefault(
+          context.getDoc().getDocumentReference());
+      try {
+        addAllExtJSfilesFromDoc(getModelAccess().getDocument(
+            getObjectPageTypeUtils().getDocRefForPageType(pageTypeRef)));
+      } catch (DocumentNotExistsException exp) {
+        LOGGER.error("Could not get Document with docRef {} ",
```

Comment:
change logging level to info. For PageTypes defined in central db or even in java no page type document does exist on a regular basis. If no document exists no object for javascript files can exist, thus it can regularly scipt.